articles/other/other-how-use-pricing-tool.md

```diff
@@ -0,0 +1,118 @@
+
+3. Open your applications folder and double-click the app **G10EstimatingTool.app**. You can identify this app by the UKCloud logo.
+
+    ![Opening the app](images/pricing-estimator-mac-install-4.png)
+
+## Using the tool
+
+Both the Mac and Windows versions of the tool operate the same way. This guide shows the Windows version, but the MacOS version is identical.
+
+1. Open the tool and click **Add** to begin your estimate.
+
+    ![Summary screen](images/pricing-estimator-1.png)
+
+2. In the newly opened window, select the product or service you want to price using the dropdown menu.
+
+    ![Product selection](images/pricing-estimator-2.png)
+
+3. Choose the various features you want to include in your estimate using the dropdown menus provided.
+
+    > [!NOTE]
+    > The more detail you select, the more accurate the estimate will be. The tool will automatically update the monthly and yearly estimates as you select the various options.
+
+    ![UKCloud for VMware](images/pricing-estimator-3.png)
+
+4. Any invalid selections or combinations will result in a dialog box appearing stating why this selection is invalid. The tool will then be returned to its state prior to the selection that caused this error message to appear.
+
+    ![Error dialog box](images/pricing-estimator-4.png)
+
+5. You can add descriptions to your estimates by entering information in the **Additional Description** field.
+
+    This will add the description within the line item and you can use it as a label as you build up your estimate package. This is all reflected in the PDF the tool can produce.
+
+    ![UKCloud for VMware](images/pricing-estimator-3.png)
+
+6. When you're done, click **Save**. The estimate will be displayed in the original window.
+
+    ![Saved dialog box](images/pricing-estimator-5.png)
+
+7. Your estimates will be displayed the next time you open the tool. You can use the **Edit** button to make changes to existing estimates.
+
+    ![Summary screen with saved estimates](images/pricing-estimator-6.png)
+
+8. The tool also provides a print function. In the menu, select **File** then **Print** to print your estimates. If a PDF printer is installed, you can print estimates in PDF format.
+
+    ![Print option](images/pricing-estimator-7.png)
+
+    The following image provides an example of how the printed version will look.
+
+    ![PDF Printout](images/pricing-estimator-8.png)
+
+9. You can copy estimates to Excel or other office software, by right-clicking the estimate and selecting **Copy All to Clipboard**. You can then paste the information from the clipboard into your desired software.
+
+    ![Copy to clipboard](images/pricing-estimator-9.png)
+
+    The following image provides an example of an estimate copied into an Excel worksheet.
+
+    ![Pasted into Excel](images/pricing-estimator-10.png)
+
+## Related videos
+
+- [UKCloud Pricing Estimator](https://vimeo.com/300701961)
```

Comment:
Should we link to the KC Video Article or the video Directly?

Reply:
I think analytics wise, we might benefit from linking the KC article. But I don't mind.

Reply:
I would recommend pointing directly to the video rather than the article, otherwise customers will have to click twice to see the video. If the article had more information, rather than just a link to the video (or if we could embed the streaming video directly in the article (which we can't)), it might benefit customers to go to the article first, but as it doesn't I think it would just be annoying!

Reply:
Sure, 100% understand that.